src/nwp_consumer/internal/inputs/ecmwf/README.md

```diff
@@ -0,0 +1,16 @@
+# ECMWF API
+
+## Authentication
+
+The ECMWF API requires the setting of a few environment variables,
```

Comment:
could you list the `environment variables`

Reply:
They are, in `config.yaml` which this says to refer to - I thought it was better just have them just in one place in case in multiple places they forget to be updated and end up out of date? But if you think it's not clear enough to look there on on PyPi I can add them in.

Reply:
Yea I would just put them in here, and then have to remember to update this readme and the config file if we add a new one